backend/src/main/java/com/cs471/studentLoanSystem/login/LoginResponse.java

```diff
@@ -1,54 +0,0 @@
-package com.cs471.studentLoanSystem.login;
```

Comment:
What's the purpose of maintaining two different login responses?

This should be able to do both? One of either `bankInfo` or `studentInfo` will be null. (which is fine)

Reply:
What prompted the different responses was largely the registrar role. That role requires only whether it's authenticated or not and what role they are. I couldn't have a name in that response, so I broke the registrar response from the typical response. At that point, I had already made a separate response, so I followed suit with the student and officer response, breaking them up further. I also think this is proper for if we want to extend the student/officer response class to include more information that may become relevant, as we don't want to clutter a response with unnecessary fields.

Reply:
I think it would be useful if there was a name in the registrar response, even if you just feed the username back. The name is used in the frontend.